packages/esm-patient-medications-app/src/config-schema.ts

```diff
@@ -41,3 +41,11 @@ export interface ConfigObject {
   showPrintButton: boolean;
   maxDispenseDurationInDays: number;
 }
+
+export default {
+  debounceTime: {
+    type: 'integer',
+    default: 500,
+    description: 'Debounce time for medication search input in milliseconds',
+  },
```

Comment:
This block of config is in the wrong place. It should exist inside the `configSchema` object. Also, the object's keys don't match the schema format. It should read as follows:

```suggestion
export const configSchema = {
  // existing config schema properties...
  drugSearchDebounceTimeInMs: {
    _type: 'integer',
    _description: 'Number of milliseconds to delay the search operation in the drug search input by after the user starts typing. The useDebounce hook delays the search by 300ms by default',
    _default: 300,
  }
}
```

And then the new `debounceTimeInMs` configuration property should be added to the `ConfigObject` interface as shown below:
```suggestion
  debounceTime: {
    type: 'integer',
    default: 500,
    description: 'Debounce time for medication search input in milliseconds',
  },
```

```tsx
export interface ConfigObject {
  daysDurationUnit: {
    uuid: string;
    display: string;
  };
  drugOrderTypeUUID: string;
  showPrintButton: boolean;
  maxDispenseDurationInDays: number;
  drugSearchDebounceTimeInMs: number;
}
```

Reply:
@denniskigen thanks so so much for the guidance. Am so glad I have learnt something new